### PR TITLE
Add missing licences

### PR DESCRIPTION
**Pull request by claudiu-stanciu**
Master branch build is KO without adding the licences

### CODE REVIEW

**Comment by claudiu-stanciu** [comment]
I need to fix offline the merge issue. Not sure what github did here ...

File: services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java [hunk 1]
```diff
@@ -19,6 +19,26 @@
  * #L%
  */
 
+/*-
+ * #%L
+ * thinkbig-feed-manager-core
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (KyloTech) [inline services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by KyloTech** [comment]
SonarQube analysis reported 10 issues
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

#### 9 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SimpleCategoryCache.java#L8](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L8): This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [SimpleCategoryCache.java#L201](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L201): Call "Optional#isPresent()" before accessing the value. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)
1. ![MAJOR][MAJOR] [CategoryFeedService.java#L9](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/operational-metadata-service/operational-metadata-integration-service/src/main/java/com/thinkbiganalytics/metadata/cache/CategoryFeedService.java#L9): This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MINOR][MINOR] [SimpleCategoryCache.java#L112](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L112): Immediately return this expression instead of assigning it to the temporary variable "feedCategory". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [SimpleCategoryCache.java#L130](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L130): Immediately return this expression instead of assigning it to the temporary variable "feedCategory". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [SimpleCategoryCache.java#L146](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L146): Immediately return this expression instead of assigning it to the temporary variable "list". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [SimpleCategoryCache.java#L149](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L149): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [SimpleCategoryCache.java#L150](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java#L150): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [CategoryFeedService.java#L50](https://github.com/Teradata/kylo/blob/5f71b9d9b16f3804e2ccbe5e93bc85e2200aa098/services/operational-metadata-service/operational-metadata-integration-service/src/main/java/com/thinkbiganalytics/metadata/cache/CategoryFeedService.java#L50): Immediately return this expression instead of assigning it to the temporary variable "feedCategoryListMap". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by claudiu-stanciu** [comment]
Closing as my somehow my repository didn't pick-up all the latest commits ....

**Pull request closed without merging**